indico/modules/events/persons/placeholders.py

```diff
@@ -63,6 +64,27 @@ def render(cls, person, event, **kwargs):
                                                                           title=event.title)
 
 
+class EventContributionsPlaceholder(ParametrizedPlaceholder):
+    name = 'event_contributions'
+    description = _("Event contributions, optional: One where speaker")
+    param_friendly_name = 'onlyspeakers'
+    param_required = False
+
+    @classmethod
+    def render(cls, param, person, event, **kwargs):
+        res = "<ul>\n"
+        for c in get_contributions_for_person(event, person, (param and param == 'onlyspeakers')):
+            url = url_for('contributions.display_contribution', c)
+            res += "<li> <a href=\"{url}\">{title} </a>".format(url=url, title=c.title)
```

Comment:
that looks like an XSS risk. better use a Jinja template for this where autoescaping is used.

Reply:
Done